<!DOCTYPE html>
<html lang="" xml:lang="">
  <head>
    <title>DNA Barcoding of Fern Gametophytes: Past, Present, and Future</title>
    <meta charset="utf-8" />
    <meta name="author" content="Joel H. Nitta" />
    <meta name="date" content="2022-03-18" />
    <script src="index_files/header-attrs/header-attrs.js"></script>
    <link rel="stylesheet" href="xaringan-themer.css" type="text/css" />
  </head>
  <body>
    <textarea id="source">

class: hide-count















&lt;br&gt;
# DNA Barcoding of Fern Gametophytes:&lt;br&gt;Past, Present, and Future

### Joel H. Nitta  
University of Tokyo


XVI Conference of the&lt;br&gt;Indian Fern Society  
2022.03.18

&lt;img src="images/title_back.png" height="200" style="position:absolute; right:80px; top:300px;"&gt;

---

## Preprint

- #### Nitta and Chambers 2022 *Apps. in Plant Sci.* (in press)&lt;br&gt;https://ecoevorxiv.org/dr25p/

## Slides

- ### https://joelnitta.github.io/gameto_barcode_ifs

## More Info

- ### https://joelnitta.com

---
class: center, middle

## Goal: Provide a practival overview of&lt;br&gt;DNA barcoding in fern gametophytes

---

class: inverse, center, middle

# What is DNA barcoding?

---

## What is DNA barcoding?

.pull-left[
Use of one\* DNA locus for identifying species (Hebert, et al., 2003; Hebert, et al., 2003)
]
&lt;!-- [@Hebert2003; @Hebert2003a] --&gt;

.pull-right[
&lt;img src="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-021-81087-w/MediaObjects/41598_2021_81087_Fig10_HTML.png?as=webp" height="260"&gt;
&lt;br&gt;&lt;br&gt;
&lt;img src="images/orchid_barcodes.png" height="110"&gt;
&lt;br&gt;.small[Li, et al. (2021)]
]

---

## What is DNA barcoding?

.pull-left[
Use of one\* DNA locus for identifying species (Hebert, et al., 2003; Hebert, et al., 2003)

- __"Barcode" is a misnomer__
  - No locus is identical across all individuals of a species and different between different species
  
- Mitochondrial *COI* is used in animals

- \*No single marker available in plants
]

.pull-right[
&lt;img src="https://innovativegenomics.org/wp-content/uploads/2018/04/DNA.png" height="140"&gt; .xxl[≠] &lt;img src="https://upload.wikimedia.org/wikipedia/commons/8/84/EAN13.svg" height="140"&gt;
]

---



## Why DNA barcoding of ferns?

--

### 1. Primary taxonomy

--

DNA sequences provide __objective evidence of species status__

Mean infrageneric, interspecific *rbcL* distance across ferns: __1.50% ± 0.78%__  
(*n* = 4,711 species, 259 genera; Nitta, in prep.)

.footnote[mean ± SD]

---

## Why DNA barcoding of ferns?

### 1. Primary taxonomy

DNA sequences provide __objective evidence of species status__

Mean infrageneric, interspecific *rbcL* distance across ferns: __1.50% ± 0.78%__  
(*n* = 4,711 species, 259 genera; Nitta, in prep.)

### Phylogenetic analysis can make new species descriptions __more robust__

.footnote[mean ± SD]

---

## Why DNA barcoding of ferns?

### 2. Identify field-collected gametophytes

.center[&lt;img src="images/sessa_life_cycle.jpg" height="400"&gt;]

---

## Why DNA barcoding of ferns?

### Knowlege of gametophyte ecology is practically nil compared to sporophytes

.center[&lt;img src="images/sessa_life_cycle_question.jpg" height="400"&gt;]

---



## Why DNA barcoding of ferns?

.pull-left[

&gt; "In combination these aspects of __prothallial morphology serve to characterize most of the larger groups of homosporous ferns__, nearly as clearly as sporophyte morphology"  
.small[Nayar and Kaur (1971)]

Gametophyte morphology is important for systematics, but __cannot be relied on to consistently identify species__

]

.pull-right[
.center[
&lt;img src="images/nayar_kaur_1971.png" height="400"&gt;
]
]



---

## Why DNA barcoding of ferns?

.pull-left[
A, B: __Cordate__ (many terrestrial species)  

C, D: __Ribbon__ (e.g., Vittariaceae, Hymenophyllaceae)  

E, F: __Filamentous__  (e.g., Schizaeaceae, Hymenophyllaceae) 
]

&lt;img src="images/nitta_2022_1.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## Why DNA barcoding of ferns?

.pull-left[
__Cordate__  
A: *Sphaeropteris medullaris* (G. Forst.) Bernh.  
B: *Austroblechnum raiateense* (J.W.Moore) Gasper
&amp; V.A.O.Dittrich

__Ribbon__  
C: *Callistopteris apiifolia* (C. Presl) Copel.  
D: *Hymenophyllum polyanthos* (Sw.) Sw.

__Filamentous__  
E: *Crepidomanes minutum* (Blume) K.Iwats.  
F: *Abrodictyum dentatum*(Bosch) Ebihara &amp; K.Iwats.
]

&lt;img src="images/nitta_2022_1.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## The beginning: ID of a single gametophyte

.pull-left[
- First use of DNA barcodes in ferns
- Sequence *rbcL* from gametophyte in culture, query GenBank
- Identification as *Osmunda*

.footnote[Schneider, et al. (2006)]
]

.pull-right[
.center[
&lt;img src="images/osmunda_gameto.png" height="200"&gt;

.small[Soare (2008)]
]
]

---

## Typical approach

.center[
&lt;img src="images/barcode_flow.png" height="450"&gt;
]

---

## Typical approach

.center[
&lt;img src="images/barcode_flow_2.png" height="450"&gt;
]

---

## Choosing a marker

### My recommendation: *rbcL* + *trnL-trnF*

- Nuclear markers are too difficult to obtain (multiple copies)
- *rbcL* can differentiate between species in most cases, has best coverage on GenBank
- *trnL-F* can be used as secondary marker for closely related taxa

| Marker | Type | PCR success | Variability |
| ------ | ---- | ----------- | ----------- |
| __*rbcL(-a)*__ | Coding | High | Low |
| *matK* | Coding | Low | Moderate to high
| *trnH-psbA* | Spacer | High | High in some groups, low in others
| __*trnL-F*__ | Spacer | High | High

---
class: inverse, center, middle

# Building the library

---

## Building the library

Sanger sequencing of (at least) one specimen per species in study area from sporophytes

---

## Building the library

Sanger sequencing of (at least) one specimen per species in study area from sporophytes

.pull-left[

What about __multiple individuals per species?__
- Needed to assess __"barcode gap"__
- For __*rbcL*__, almost certain to be __zero variation__

]

.pull-right[
.center[&lt;img src="images/paulay_2005_1.png" height="350"&gt;]
.small[Meyer, et al. (2005)]
]

---

## Building the library

Sanger sequencing of (at least) one specimen per species in study area from sporophytes

.pull-left[

What about __multiple individuals per species?__
- Needed to assess __"barcode gap"__
- For __*rbcL*__, almost certain to be __zero variation__

__My recommendation:__
- __One specimen/species for most taxa__
- Multiple specimens in case of "difficult" taxa (species complexes)
]

.pull-right[
.center[&lt;img src="images/paulay_2005_1.png" height="350"&gt;]
.small[Meyer, et al. (2005)]
]

---
## Case study 1:&lt;br&gt;Pteridophytes of Japan

.pull-left[
- *rbcL* + *trnH-psbA*

- 733 taxa, 1 individual per species

- High success in sexual diploids, lower in polyploid or apogamous taxa

.footnote[Ebihara, et al. (2010)]

]

&lt;img src="images/ebihara_2010_1.png"
     height="600px"
     style="position:absolute; right:30px; top:30px;"&gt;

---

## Case study 2: Ferns of Moorea and Tahiti, French Polynesia

.pull-left[
- *rbcL* + *trnH-psbA*

- 145 spp., 1 individual per species for most

- High success rate overall (better than Japan)

.footnote[Nitta, et al. (2017)]
]

.pull-right[
.center[&lt;img src="images/nitta_2017_1.png" height="300"&gt;]
]

---
class: inverse, center, middle

# Using the barcode

---

## Case study 1: Independent gametophytes in Japan
&lt;br&gt;
.middle[.center[&lt;img src="images/indep_gameto.png" height="450"&gt;]]

---

## Case study 1: Independent gametophytes in Japan

__Focus on gametophyte mats__

- *Hymenophyllum mikawanum* (Seriz.) Seriz.:&lt;br&gt; sporophyte endangered
- *Haplopteris mediosora* (Hayata) X.C.Zhang:&lt;br&gt; sporophyte extremely rare
- *Antrophyum plantagineum* (Cav.) Kaulf:&lt;br&gt; sporophyte unknown in Japan
  
.footnote[Murakami, et al. (2021)]

&lt;img src="images/murakami_2021_1.png" height="230" style="position:absolute; right:130px; top:160px;"&gt;
&lt;img src="images/murakami_2021_2.png" height="220" style="position:absolute; right:130px; top:430px;"&gt;

---

## Case study 1: Independent gametophytes in Japan

Focus on gametophyte mats

- *Hymenophyllum mikawanum* (Seriz.) Seriz.:&lt;br&gt; __sporophyte endangered__
- *Haplopteris mediosora* (Hayata) X.C.Zhang:&lt;br&gt; __sporophyte extremely rare__
- *Antrophyum plantagineum* (Cav.) Kaulf:&lt;br&gt; __sporophyte unknown in Japan__
  
.footnote[Murakami, et al. (2021)]

&lt;img src="images/murakami_2021_1.png" height="230" style="position:absolute; right:130px; top:160px;"&gt;
&lt;img src="images/murakami_2021_2.png" height="220" style="position:absolute; right:130px; top:430px;"&gt;

---

## Case study 1: Independent gametophytes in Japan

Focus on gametophyte mats

- *Hymenophyllum mikawanum* (Seriz.) Seriz.:&lt;br&gt; sporophyte endangered
- __*Haplopteris mediosora*__ (Hayata) X.C.Zhang:&lt;br&gt; sporophyte extremely rare
- *Antrophyum plantagineum* (Cav.) Kaulf:&lt;br&gt; sporophyte unknown in Japan
  
.footnote[Murakami, et al. (2021)]

&lt;img src="images/murakami_2021_3.png" height="400" style="position:absolute; right:30px; top:160px;"&gt;

---

## Case study 2: Gametophyte community surveys in Japan

.pull-left[
- First use of __garden net__ for sampling
- Non-cordate gametophytes tend to occur separate from sporophytes
- Identify several new independent gametophytes

.footnote[Ebihara, et al. (2013)]
]

.pull-right[
.center[&lt;img src="images/ebihara_2013_1.png" height="350"&gt;]
]

---

## Case study 2: Gametophyte community surveys in Japan

.pull-left[
- First use of __garden net__ for sampling
- Non-cordate gametophytes tend to occur separate from sporophytes
- Identify several new independent gametophytes

.footnote[Ebihara, et al. (2013)]
]

.pull-right[
.center[
&lt;img src="images/ebihara_2013_2.png" height="150"&gt;
&lt;img src="images/ebihara_2013_3.png" height="150"&gt;
]
]

---

## Case study 3: Community structure of ferns in Tahiti

.pull-left[
- __96-well plates for DNA extraction, PCR__
- Sporophytes are more affected by environment
- Gametophytes are widely distributed, but observed fewer than expected

.footnote[Nitta, et al. (2017)]
]

.pull-right[
.center[&lt;img src="images/plates.jpg" height="380"&gt;]
]

---

## Case study 3: Community structure of ferns in Tahiti

.pull-left[
- 96-well plates for DNA extraction, PCR
- __Sporophytes are more affected by environment__
- Gametophytes are widely distributed, but observed fewer than expected

.footnote[Nitta, et al. (2017)]
]

&lt;img src="images/nitta_2017_2.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## Case study 4: Hemi-epiphytism in *Vandenboschia*

.pull-left[
- Unclear if *V. collariata* was __primary__ or __secondary__ hemi-epiphyte

.footnote[Nitta, et al. (2009)]
]

.pull-right[
.center[&lt;img src="images/nitta_2009_1.png" height="450"&gt;]
]

---

## Case study 4: Hemi-epiphytism in *Vandenboschia*

.pull-left[
- Unclear if *V. collariata* was primary or secondary hemi-epiphyte
- Find gametophytes at __base of tree__
- Sequence DNA to confirm identity
- ➡︎ Shows *V. collariata* is __primary hemiepiphyte__

.footnote[Nitta, et al. (2009)]
]

.pull-right[
.center[&lt;img src="images/nitta_2009_2.png" height="230"&gt;]
.center[&lt;img src="images/nitta_2009_3.png" height="230"&gt;]
]

---

## Case study 5: Desiccation tolerance of filmy ferns

.pull-left[
- DT is important trait in __transition of plants to life on land__
- DT is known from both sporophytes and gametophytes of __filmy ferns__
- Compare DT between sporophytes and gametophytes

.footnote[Nitta, et al. (2021)]
]

&lt;img src="images/nitta_2021_1.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## Case study 5: Desiccation tolerance of filmy ferns

&lt;img src="images/nitta_2021_2_trans.png" height="520" style="position:absolute; right:80px; top:140px;"&gt;

- Gametophytes have __less DT than sporophytes__

.footnote[Nitta, et al. (2021)]


---

## Case study 5: Desiccation tolerance of filmy ferns

&lt;img src="images/nitta_2021_3.png" height="350" style="position:absolute; right:140px; top:240px;"&gt;

- Perhaps filmy ferns rely on __gemmae and microhabitats__, not DT

.footnote[Nitta, et al. (2021)]

---
class: inverse, center, middle

# The future: Next-generation DNA sequencing

---

## Disadvantages of Sanger sequencing

- Slow
- Limited number of samples
- Limited number of loci

High-throughput methods could allow for **continuous monitoring over time and space** of gametophyte populations

.center[&lt;img src="https://www.pacb.com/wp-content/uploads/Evolution-of-sequencing-technology.jpg" height="260"&gt;]

.footnote[https://www.pacb.com/]

---

## Next-Gen DNA sequencing: Microfluidic PCR

.pull-left[
- Allows to __massively scale-up sequencing__
- Expensive

.footnote[Gostel, et al. (2020)]
]

&lt;img src="images/gostel_2020_1.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## Next-Gen DNA sequencing: MinION

.pull-left[
- Portable DNA sequencer
- Long reads (ca. 1,000 bp)
- Enables __identification of species&lt;br&gt;in the field__

.footnote[Pomerantz, et al. (2018)]
]

&lt;img src="images/pomerantz_2018_1.png" height="500" style="position:absolute; right:80px; top:140px;"&gt;

---

## Tissue-direct PCR

.pull-left[
- __Skips DNA extraction step__
- Used to survey *Lomariopsis* in Taiwan
- Possible to combine with next-gen sequencing?

.footnote[Li, et al. (2009); Wu, et al. (2022)]
]

.pull-right[
.center[&lt;img src="images/kuo_2022_3.png" height="350"&gt;]
.small[Photo: L.-Y. Kuo]
]

---

## Tissue-direct PCR

.pull-left[
- Skips DNA extraction step
- __Used to survey *Lomariopsis* in Taiwan__
- Possible to combine with next-gen sequencing?

.footnote[Li, et al. (2009); Wu, et al. (2022)]
]

&lt;img src="images/kuo_2022_2.png" height="360" style="position:absolute; right:30px; top:140px;"&gt;

---

## Conclusions

- DNA barcoding can provide unprecedented insights into fern biology
- Sanger sequencing is (still) useful, but limited
- Next-generation DNA barcoding has the potential to revolutionize fern biology (again)

.center[&lt;img src="images/nitta_2022_2.png" height="380"&gt;]

---

## Acknowledgements

- Co-author Sally Chambers
- Marie Selby Botanical Garden
- Li-Yaung Kuo
- Organizing Committee of the XVI Conference of the Indian Fern Society

---

## References

.small[
Ebihara, A. et al. (2010). "Molecular species identification with rich
floristic sampling: DNA barcoding the pteridophyte flora of Japan". In:
_PLoS ONE_ 5.12, p. e15136. DOI:
[10.1371/journal.pone.0015136](https://doi.org/10.1371%2Fjournal.pone.0015136).

Ebihara, A. et al. (2013). "A survey of the fern gametophyte flora of
Japan: Frequent independent occurrences of noncordiform gametophytes".
In: _American Journal of Botany_ 100.4, pp. 735-743. DOI:
[10.3732/ajb.1200555](https://doi.org/10.3732%2Fajb.1200555).

Gostel, M. R. et al. (2020). "Microfluidic Enrichment Barcoding
(MEBarcoding): A New Method for High Throughput Plant DNA Barcoding".
In: _Scientific Reports_ 10.1 (1), p. 8701. DOI:
[10/gg539d](https://doi.org/10%2Fgg539d).

Hebert, P. et al. (2003). "Ten Species in One: DNA Barcoding Reveals
Cryptic Species in the Neotropical Skipper Butterfly Astraptes
Fulgerator". In: _Proceedings of the National Academy of Sciences of
the United States of America_ 101, pp. 114812-14817.
]

---

## References (cont.)

.small[
Hebert, P. et al. (2003). "Barcoding Animal Life: Cytochrome _c_
Oxidase Subunit 1 Divergences among Closely Related Species". In:
_Proceedings of the Royal Society of London, Series B: Biological
Sciences_ 270, pp. S96-S99.

Li, F. et al. (2009). "Tissue Direct PCR, a Rapid and Extraction Free
Method for Barcoding of Ferns". In: _Molecular Ecology Resources_ 10.1,
pp. 92-95.

Li, H. et al. (2021). "The Specific DNA Barcodes Based on Chloroplast
Genes for Species Identification of Orchidaceae Plants". In: _Sci Rep_
11.1, p. 1424. DOI:
[10.1038/s41598-021-81087-w](https://doi.org/10.1038%2Fs41598-021-81087-w).

Meyer, C. P. et al. (2005). "DNA Barcoding: Error Rates Based on
Comprehensive Sampling". In: _PLoS Biology_ 3.12, p. e422.
]

---

## References (cont.)

.small[
Murakami, N. et al. (2021). "Inventories of fern independent
gametophytes in Japan using DNA barcoding based on nucleotide sequences
of _rbcL_ gene (in Japanese)". In: _BSJ-Review_ 12, pp. 159-168. DOI:
[10.24480/bsj-review.12c3.00211](https://doi.org/10.24480%2Fbsj-review.12c3.00211).

Nayar, B. K. et al. (1971). "Gametophytes of Homosporous Ferns". In:
_The Botanical Review_ 37.3, pp. 295-396.

Nitta, J. H. et al. (2009). "Hemi-epiphytism in _Vandenboschia
collariata_ (Hymenophyllaceae)". In: _Brittonia_ 61.4, pp. 392-397.
DOI:
[10.1007/s12228-009-9097-5](https://doi.org/10.1007%2Fs12228-009-9097-5).

Nitta, J. H. et al. (2017). "Life cycle matters: DNA barcoding reveals
contrasting community structure between fern sporophytes and
gametophytes". In: _Ecological Monographs_ 87.2, pp. 278-296. DOI:
[10.1002/ecm.1246](https://doi.org/10.1002%2Fecm.1246).
]

---

## References (cont.)

.small[
Nitta, J. H. et al. (2021). "Ecophysiological differentiation between
life stages in filmy ferns (Hymenophyllaceae)". In: _J Plant Res_ 134,
pp. 971-988. DOI:
[10.1007/s10265-021-01318-z](https://doi.org/10.1007%2Fs10265-021-01318-z).

Pomerantz, A. et al. (2018). "Real-Time DNA Barcoding in a Rainforest
Using Nanopore Sequencing: Opportunities for Rapid Biodiversity
Assessments and Local Capacity Building". In: _GigaScience_ 7.4, p.
giy033. DOI: [10/gc84n7](https://doi.org/10%2Fgc84n7).

Schneider, H. et al. (2006). "Identifying Fern Gametophytes Using DNA
Sequences". In: _Molecular Ecology Notes_ 6, pp. 989-991. DOI:
[10.1111/j.1471-8286.2006.01424.x](https://doi.org/10.1111%2Fj.1471-8286.2006.01424.x).

Soare, L. C. (2008). "In Vitro Development of Gametophyte and
Sporophyte in Several Fern Species". In: _Notulae Botanicae Horti
Agrobotanici Cluj-Napoca_ 36. DOI:
[10.15835/nbha36183](https://doi.org/10.15835%2Fnbha36183).

Wu, Y. et al. (2022). "Integrating Tissue-Direct PCR into Genetic
Identification: An Upgraded Molecular Ecology Approach to Survey Fern
Gametophytes in the Field". In: _Applications in Plant Sciences_ 10.2,
p. e11462.
]
    </textarea>
<style data-target="print-only">@media screen {.remark-slide-container{display:block;}.remark-slide-scaler{box-shadow:none;}}</style>
<script src="https://remarkjs.com/downloads/remark-latest.min.js"></script>
<script>var slideshow = remark.create({
"slideNumberFormat": "%current%",
"highlightStyle": "github",
"highlightLines": true,
"ratio": "16:9",
"countIncrementalSlides": true
});
if (window.HTMLWidgets) slideshow.on('afterShowSlide', function (slide) {
  window.dispatchEvent(new Event('resize'));
});
(function(d) {
  var s = d.createElement("style"), r = d.querySelector(".remark-slide-scaler");
  if (!r) return;
  s.type = "text/css"; s.innerHTML = "@page {size: " + r.style.width + " " + r.style.height +"; }";
  d.head.appendChild(s);
})(document);

(function(d) {
  var el = d.getElementsByClassName("remark-slides-area");
  if (!el) return;
  var slide, slides = slideshow.getSlides(), els = el[0].children;
  for (var i = 1; i < slides.length; i++) {
    slide = slides[i];
    if (slide.properties.continued === "true" || slide.properties.count === "false") {
      els[i - 1].className += ' has-continuation';
    }
  }
  var s = d.createElement("style");
  s.type = "text/css"; s.innerHTML = "@media print { .has-continuation { display: none; } }";
  d.head.appendChild(s);
})(document);
// delete the temporary CSS (for displaying all slides initially) when the user
// starts to view slides
(function() {
  var deleted = false;
  slideshow.on('beforeShowSlide', function(slide) {
    if (deleted) return;
    var sheets = document.styleSheets, node;
    for (var i = 0; i < sheets.length; i++) {
      node = sheets[i].ownerNode;
      if (node.dataset["target"] !== "print-only") continue;
      node.parentNode.removeChild(node);
    }
    deleted = true;
  });
})();
(function() {
  "use strict"
  // Replace <script> tags in slides area to make them executable
  var scripts = document.querySelectorAll(
    '.remark-slides-area .remark-slide-container script'
  );
  if (!scripts.length) return;
  for (var i = 0; i < scripts.length; i++) {
    var s = document.createElement('script');
    var code = document.createTextNode(scripts[i].textContent);
    s.appendChild(code);
    var scriptAttrs = scripts[i].attributes;
    for (var j = 0; j < scriptAttrs.length; j++) {
      s.setAttribute(scriptAttrs[j].name, scriptAttrs[j].value);
    }
    scripts[i].parentElement.replaceChild(s, scripts[i]);
  }
})();
(function() {
  var links = document.getElementsByTagName('a');
  for (var i = 0; i < links.length; i++) {
    if (/^(https?:)?\/\//.test(links[i].getAttribute('href'))) {
      links[i].target = '_blank';
    }
  }
})();
// adds .remark-code-has-line-highlighted class to <pre> parent elements
// of code chunks containing highlighted lines with class .remark-code-line-highlighted
(function(d) {
  const hlines = d.querySelectorAll('.remark-code-line-highlighted');
  const preParents = [];
  const findPreParent = function(line, p = 0) {
    if (p > 1) return null; // traverse up no further than grandparent
    const el = line.parentElement;
    return el.tagName === "PRE" ? el : findPreParent(el, ++p);
  };

  for (let line of hlines) {
    let pre = findPreParent(line);
    if (pre && !preParents.includes(pre)) preParents.push(pre);
  }
  preParents.forEach(p => p.classList.add("remark-code-has-line-highlighted"));
})(document);</script>

<script>
slideshow._releaseMath = function(el) {
  var i, text, code, codes = el.getElementsByTagName('code');
  for (i = 0; i < codes.length;) {
    code = codes[i];
    if (code.parentNode.tagName !== 'PRE' && code.childElementCount === 0) {
      text = code.textContent;
      if (/^\\\((.|\s)+\\\)$/.test(text) || /^\\\[(.|\s)+\\\]$/.test(text) ||
          /^\$\$(.|\s)+\$\$$/.test(text) ||
          /^\\begin\{([^}]+)\}(.|\s)+\\end\{[^}]+\}$/.test(text)) {
        code.outerHTML = code.innerHTML;  // remove <code></code>
        continue;
      }
    }
    i++;
  }
};
slideshow._releaseMath(document);
</script>
<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
(function () {
  var script = document.createElement('script');
  script.type = 'text/javascript';
  script.src  = 'https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML';
  if (location.protocol !== 'file:' && /^https?:/.test(script.src))
    script.src  = script.src.replace(/^https?:/, '');
  document.getElementsByTagName('head')[0].appendChild(script);
})();
</script>
  </body>
</html>
